src/components/chips/chips.spec.js

```diff
@@ -0,0 +1,133 @@
+describe('<md-chips>', function() {
+
+  beforeEach(module('material.components.chips'));
+
+  function compile (str, scope) {
+    var container;
+    inject(function ($compile) {
+      container = $compile(str)(scope);
+      scope.$apply();
+    });
+    return container;
+  }
+
+  function createScope () {
+    var scope;
+    var items = ['Apple', 'Banana', 'Orange'];
+    inject(function ($rootScope) {
+      scope = $rootScope.$new();
+      scope.items = items;
+    });
+    return scope;
+  }
+
+  describe('basic functionality', function () {
+    it('should render a default input element', function() {
+      var scope = createScope();
```

Comment:
I'm not a fan of aligning assignment operators, but especially when it's not done consistently